README.md

```diff
@@ -1 +1,116 @@
-# wavefront-sdk-docs
+# Wavefront SDK Documentation Sources
+
+Wavefront supports a suite of open source SDKs that developers can use to instrument applications for observability. The instrumented application collects and sends metrics, histograms, and/or trace data to Wavefront for storage and visualization.
+
+## Table of Content
+* [What's in this repository](#What's-in-this-repository)
+* [Wavefront SDKs](#Wavefront-SDKs)
+* [License](#License)
+* [How to Contribute](#How-toContribute)
+
+## What's in this repository
+Welcome to the repository that contains documentation common to the Wavefront's SDKs! 
```

Comment:
Welcome to the repository that contains documentation common to the Wavefront's SDKs! 
=> This repository contains documentation common to several Wavefront SDKs. We expect that you'll read this information as part of learning about one of the SDKs. It's not meant for standalone consumption.